modules/EnsEMBL/Web/Query/Availability/Gene.pm

```diff
@@ -44,6 +44,19 @@ sub fixup {
   $self->SUPER::fixup();
 }
 
+sub _count_homologues {
+  my ($self, $member, $args) = @_;
+
+  my $dbc   = $self->database_dbc($args->{'species'},'compara');
+  my $gmID  = $member->gene_member_id;
+  my $sql   = 'select count(*) from homology_member where gene_member_id = ?';
+  my $sth   = $dbc->prepare($sql);
+  $sth->execute($gmID);
+
+  my @results = @{$sth->fetchall_arrayref||[]};
```

Comment:
Should this line be like this:
`@{$sth->fetchall_arrayref} ||[]` 

feels wrong to dereference an empty array. Also the return line below might produce an error if the array is empty.  can we check if this line is returning the right result?

Reply:
We should allow for a gene having no homologues, but you're right, that line will fail if the array is empty!